src/System.Diagnostics.PerformanceCounter/src/System/Diagnostics/PerformanceData/CounterType.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace System.Diagnostics.PerformanceData
+{
+    /// <summary>
+    /// Enum of friendly names to counter types (maps directory to the native types defined in winperf.h).
+    /// </summary>    
+    public enum CounterType
+    {
```

Comment:
Are these the same as the ones you added in PerformanceCounters? If so (and if different from PerformanceCounterType/Interop.PerformanceCounterOptions, can we consolidate them somewhere into something shared so we don't have duplicate code?

Reply:
Do you mean add "new counters type"(counters that are not present PerformanceCounterType.cs) in  to Interop.PerformanceCounterOptions.cs and refers to that on enums?